ehrql/docs/schemas.py

```diff
@@ -1,15 +1,21 @@
+import datetime
 from collections import defaultdict
 
 from ehrql import tables
-from ehrql.query_language import BaseFrame, Series
+from ehrql.codes import BaseCode
+from ehrql.query_language import BaseFrame, PatientFrame, Series
 from ehrql.utils.module_utils import get_submodules
 
 from .common import reformat_docstring
 
 
+SORT_ORDER = {k: i for i, k in enumerate(["beta.tpp", "beta.core"])}
```

Comment:
thought: Eventually the core schema should probably be first, then the more specific schemas. But seeing as TPP is more fully featured at this time, I see why this is not the case now.

Reply:
Yes, I think that's exactly right.